myriad-commons/src/main/java/com/ebay/myriad/common/FrameworkUpdate.java

```diff
@@ -0,0 +1,51 @@
+package com.ebay.myriad.common;
+
+import com.google.gson.Gson;
+
+/**
+ * 
+ * Framework update from the scheduler to the executor
+ *
+ */
+public class FrameworkUpdate {
+    /**
+     * Define what kind of this update is.
+     */
+    public static enum Type {Command};
+    
+    /**
+     * Define what kind of operation or command this update is.
+     */
+    public static enum Operation {Shutdown};
+    
+    public FrameworkUpdate() {
+    }
+    
+    public FrameworkUpdate(Type type, Operation operation) {
+        setType(type);
+        setOperation(operation);
+    }
+    
+    public Type getType() {
+        return type;
+    }
+    
+    public void setType(Type type) {
+        this.type = type;
+    }
+    
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+    
+    public Operation getOperation() {
+        return operation;
+    }
+    
+    public String toString () {
+        return new Gson().toJson(this);
+    }
+    
+    private Type type;
+    private Operation operation;
+}
```

Comment:
there are several things that feel odd about this case... the first is there are 2 enums which only have 1 defined possibility.  The second is, if we assume that there will be more Types and Operations, that they are defined inside of this class.   This violates good OOP practice of the open/close principle. 

thoughts?